snabble/snabble-pay-android-sdk
# Prepare first release

### What's new?

To prepare the first release the necessary components have been adjusted or added. In the process some packages have been refactored to optimize later imports.

### How to test?
* check gradle, yml and release files for correct implementation
* deploy the project locally and verify the correct structure in ur m2 repository including correct formatted poms.
* follow the following steps for the verification:
     * set global var via terminal:  export SDK_VERSION_NAME=v1.0.0
     * deploy the project to your local repo: ./gradlew pTML 
     * include the deployed package into the demo application:
        * -implementation(project(":sdk"))
        * +implementation("io.snabble.pay:sdk:1.0.0")
     * run the application
